packages/i18n-utils/src/test/localize-url.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable no-shadow -- shadowing localizeUrl makes tests readable */
```

Comment:
Bit annoying the changes didn't get picked up as a rename, see diff here: [`4046aef` (#47570)](https://github.com/Automattic/wp-calypso/pull/47570/commits/4046aef8f4c7e4a6c1bf0b4c103fa6d01a4a5c4a)

Reply:
I have a love hate relationship with the `no-shadow` warning :|